src/components/App.tsx

```diff
@@ -2,27 +2,38 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-import { usePersistentState } from './PersistentState';
+import { usePersistentState } from '../PersistentState';
 import { useState, useEffect } from 'react';
-import './App.css';
+import { variableDefaults, staticConstants } from '../constants/constants';
+import '../App.css';
 
 function App(): JSX.Element {
-  const variableDefaults = {
-    honey: 0,
-    bees: 0,
-    costOfNextBee: 1
-  };
-
   // persistent variables
   const [honey, setHoney] = usePersistentState('honey', variableDefaults.honey);
+  const [nectar, setNectar] = usePersistentState(
+    'nectar',
+    variableDefaults.nectar
+  );
   const [bees, setBees] = usePersistentState('bees', variableDefaults.bees);
 
   // non-persistent varaibles (can be recalculated on page load)
   const [costOfNextBee, setCostOfNextBee] = useState(
     variableDefaults.costOfNextBee
   );
 
+  const honeyCost = staticConstants.NECTAR_TO_HONEY_COST;
+
   // mutators
+  const incrementNectar = () => {
```

Comment:
"increment" implies addition by one - maybe "gather" would be a better name to reflect how this will work later?